### PR TITLE
Generate global nonce (2x)

### PR DESCRIPTION
**Pull request by shargon**
Partial fix for https://github.com/neo-project/neo-gui/issues/305

### CODE REVIEW

**Comment by erikzhang** [comment]
What's the difference between this one and the old one?

**Comment by shargon** [comment]
The old one only works with transfers, if you call a contract for other things, you don't have the nonce

File: neo/Wallets/Wallet.cs [hunk 1]
```diff
@@ -347,9 +357,7 @@ public Transaction MakeTransaction(List<TransactionAttribute> attributes, IEnume
                             sb.Emit(OpCode.THROWIFNOT);
                         }
                     }
-                    byte[] nonce = new byte[8];
-                    rand.NextBytes(nonce);
-                    sb.Emit(OpCode.RET, nonce);
```
Comment (vncoelho) [inline neo/Wallets/Wallet.cs]: Was it correct to just include the `nonde` with any remark?

**Comment by shargon** [comment]
@joeqian10 could you check if this solve your problem?

**Comment by joeqian10** [comment]
@shargon Yes, it fixed the "same hash invocation transaction" problem.

**Comment by shargon** [comment]
Shall I merge it?

**Comment by shargon** [comment]
> Are you going to port this for NEO3 or no need?

No needed in neo3

**Comment by lock9** [comment]
@belane  can you be the third reviewer?

**Pull request merged**